File: src/main/java/io/github/svegon/capi/mixininterface/IKeyBinding.java
package io.github.svegon.capi.mixininterface;

import net.minecraft.client.util.InputUtil;

public interface IKeyBinding {
    InputUtil.Key boundKey();

    void boundKey(InputUtil.Key key);

    int timesPressed();

    void timesPressed(int count);

    boolean isPressedOverride();
}

File: src/main/kotlin/io/github/svegon/mclientapi/mixininterface/IKeyBinding.kt
package io.github.svegon.capi.mixininterface

import net.minecraft.client.util.InputUtil

interface IKeyBinding {
    fun boundKey(): InputUtil.Key?

    fun boundKey(key: InputUtil.Key?)

    fun timesPressed(): Int

    fun timesPressed(count: Int)

    val isPressedOverride: Boolean
}
